unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/ShowHideAnimator/ShowHideAnimator.cs

```diff
@@ -56,12 +58,20 @@ void OnHideCompleted()
         }
     }
 
-    private void SetVisibility(bool visible, TweenCallback onComplete, bool instant = false)
+    public void ShowDelayHide(float delay)
+    {
+        SetVisibility(visible: true, OnShowCompleted);
+
+        void OnShowCompleted() =>
+            SetVisibility(visible: false, null).SetDelay(delay);
+    }
+
+    private TweenerCore<float, float, FloatOptions> SetVisibility(bool visible, TweenCallback onComplete, bool instant = false)
```

Comment:
what is the purpose of returning `TweenerCore` if the usages dont need it?

Reply:
It is needed to attach DOTween delay `.SetDelay(delay)` in line 66. Since it is not that clear I will add a comment there